e2e/src/step-definitions/import-account.steps.ts

```diff
@@ -0,0 +1,39 @@
+import { Given } from '@cucumber/cucumber';
+import { expect } from 'chai';
+
+import { ImportAccountTestIds } from '../../../src/app/pages/ImportAccount/ImportAccount.test-ids';
+import { BrowserContext } from '../classes/browser-context.class';
+import { Pages } from '../page-objects';
+import { findElements, getElementText } from '../utils/search.utils';
+
+Given(/I enter second mnemonic/, async () => {
+  await Pages.ImportAccountMnemonic.enterSeedPhrase(BrowserContext.secondSeedPhrase);
+});
+
+Given(/I select (.*) tab/, async (tabName: string) => {
+  const tabElements = await findElements(ImportAccountTestIds.tabSwitcher);
```

Comment:
the [same](https://github.com/madfish-solutions/templewallet-extension/pull/852/files#r1093598035)

Reply:
done